tutorials/developer-tutorials/createNewIdentityAttributes.md

```diff
@@ -0,0 +1,168 @@
+## **Create New Identity With Attributes Using Local Fabric CA**
+`15-20 mins`
+
+## Learning Objectives
+
+* Create our smart contract
+* Create a new identity with attributes using Local Fabric CA.
+* Use the newly created identity and implement attribute-based access control.
+    * Package the smart contract
+    * Install the smart contract
+    * Instantiate the smart contract
+
+> Note: This tutorial assumes you have created the `Jack` identity by following the <a href='./createNewIdentity.md'>previous tutorial </a>.
+
+---
+<details>
+<summary><b>1. Creating our smart contract</b></summary>
+
+For the purposes of this tutorial, we'll use TypeScript as the example language.
+
+> In VS Code, every command can be executed from the Command Palette (press `Ctrl+Shift+P`, or `Cmd+Shift+P` on MacOS). All of this extension's commands start with `IBM Blockchain Platform:`. In the tutorial steps, we'll explain where to click in the UI, but look out for comment-boxes like this one if you want to know the Command Palette alternatives.
+
+1. In the left sidebar, click on the __IBM Blockchain Platform__ icon (it looks like a square, and will probably be at the bottom of the set of icons if this was the latest extension you installed!)
+
+2. Mouse-over the `SMART CONTRACTS` panel, click the `...` menu, and select `Create New Project` from the dropdown.
+
+> Command Palette alternative: `Create New Project`
+
+3. Choose a smart contract language. JavaScript, TypeScript, Java and Go are all available. For the purpose of this tutorial, please choose `TypeScript`.
+
+4. The extension will ask you if you want to name the asset in the generated contract. For the sake of this tutorial, we'll be using a `Car` as our asset.
+
+5. Choose a location to save the project.  Click `Browse`, then click `New Folder`, and name the project what you want e.g. `carContract`.
+
+> __Pro Tip:__ Avoid using spaces when naming the project!
+
+6. Click `Create` and then select the new folder you just created and click `Save`.
+
+7. Finally, select `Add to workspace` from the list of options.
+
+</details>
+
+<details>
+<summary><b>2. Creating a new identity with attributes</b></summary>
+
+In the previous tutorial (creating a new identity), we learnt how to create our own identities without giving them any attributes. In this tutorial, we will be learning how to create an identity with attributes.
```

Comment:
Maybe theses two paragraphs want to go at the top of the tutorial as an intro section